src/Actions.ts

```diff
@@ -25,7 +25,7 @@ export function getMyDetails(success?: any, error?: any) {
 
 export function filterModel(model: string, item: any, success?: any, error?: any, queryParams?: any) {
     return (dispatch: Dispatch<any>) => {
-        dispatch({ type: "FETCH_" + model + "_STARTED", model: model })
+        dispatch({ type: "FETCH_" + model + "_FILTER", model: model , item: item})
```

Comment:
keep this `_STARTED`, pass one more key to signify filtering and use that in reducer.